test/cli/autogen-autoloader/autogen-autoloader.out

```diff
@@ -166,7 +166,16 @@ Opus::Require.on_autoload('Yabba::Dabba::Jazz::JazBaz')
 class Yabba::Dabba::Jazz::JazBaz
 end
 
-Opus::Require.for_autoload(Yabba::Dabba::Jazz::JazBaz, "test/cli/autogen-autoloader/bar.rb")
+Opus::Require.autoload_map(Yabba::Dabba::Jazz::JazBaz, {
+  VALUE: "autoloader/Yabba/Dabba/Jazz/JazBaz/VALUE.rb",
+})
+
+--- output/Yabba/Dabba/Jazz/JazBaz/VALUE.rb
+# frozen_string_literal: true
+# typed: true
+
+
+Opus::Require.for_autoload(nil, "test/cli/autogen-autoloader/bar.rb")
```

Comment:
the `nil` here worries me

Reply:
Could you double-check if it's safe?

Reply:
There are thousands of lines that look similar in the `/build` folder.